stack/deprecating-interfaces.rst

```diff
@@ -146,3 +148,14 @@ Add a similar warning to the :file:`index.rst` file documenting this package (e.
 Finally, add a note to the top-level :file:`README` file in the package::
 
     *Warning:* This package is deprecated, and will be removed from the Rubin Observatory Science Pipelines distribution after release 21.0.0.
+
+
+Package Removal
+===============
+
+After deprecating a package as described `above <package-deprecation>`_, there are four steps that need to take place to actually remove the package.
+
+1. Remove the package from all eups table files that contain it.
+2. Rename the package, prefixing the string ``legacy-``, using the "Rename" button at the top of the repository settings page.
+3. Move the package to the ``lsst-dm`` GitHub organization using the "Transfer ownership" button at the bottom of the repository settings page.
+4. Edit the URL in the ``etc/repos.yaml`` file in the ``lsst/repos`` repository to correspond to the new location of the package's GitHub repository.
```

Comment:
Do these steps need to be done close together in time to avoid breaking automated processes?  E.g., all during a single day?  Or are there "trampolines" in GitHub that can absorb the transition?